iam/utils.ts

```diff
@@ -0,0 +1,77 @@
+import { iam, getCallerIdentity } from "@pulumi/aws";
+import { getCluster } from "@pulumi/aws/eks";
+import { readFileSync } from "fs";
+
+export function createPolicy(file: string, policyName: string) {
+  const policy = readFile(file);
+  return new iam.Policy(policyName, {
+    policy: policy,
+  });
+}
+
+export function createRole(
+  file: string,
+  roleName: string,
+  accountId: string,
+  clusterId: string,
+) {
+  const placeholderRole = readFile(file);
+
+  const partiallyUpdatedRole = placeholderRole.replace(
+    new RegExp("{{AWS_ACCOUNT_ID}}", "g"),
+    accountId,
+  );
+
+  const updatedRole = partiallyUpdatedRole.replace(
+    new RegExp("{{EKS_CLUSTER_ID}}", "g"),
+    clusterId,
+  );
```

Comment:
@Racer159 and I chatted about this. Thought maybe a regular string replacement could work here, but there is more than one occurrence of `{{EKS_CLUSTER_ID}}` that needs to be replaced in the roles, so the options are either regex or a templating language.

Updated to use handlebars for replacing the IAM role placeholder values instead of regex